soh/soh/Enhancements/randomizer/hook_handlers.cpp

```diff
@@ -1591,6 +1591,17 @@ void RandomizerOnVanillaBehaviorHandler(GIVanillaBehavior id, bool* should, va_l
             }
             break;
         }
+        case VB_GIVE_RANDO_GLITCH_FISHING_PRIZE: {
+            if (IS_RANDO) {
+                Fishing* fishing = va_arg(args, Fishing*);
+                if (!Flags_GetRandomizerInf(RAND_INF_ADULT_FISHING)) {
+                    Flags_SetRandomizerInf(RAND_INF_ADULT_FISHING);
+                }
+                *should = true;
+                fishData->actor->stateAndTimer = 0;
```

Comment:
Oops missed this in the suggestion
```suggestion
                fishing->actor->stateAndTimer = 0;
```

Reply:
My baguette, I saw the name change and also missed before committing